### ISSUE TITLE
Upload of files larger than 12MB doesnt work

### ISSUE THREAD

**Issue by Joly0** [bewcloud/bewcloud#6]
Hey, here i am again :)

So i just installed the docker container provided and overall things seem to work, but i noticed, that i cant upload any files larger than 12MB.

I test with smaller files between 1 and 11MB and those worked without any issues. For files between 12 and about 50MB bewcloud loads for a short moment and probably tries to upload something (without any upload indicator other than the circle thats turning its hard to say) and for any files larger than that (i tried files between 150MB and 6GB) bewcloud just loads forever and nothing happens. The small circle in the files ui just keeps loading.

So there seems to be a bug with uploading larger files, i have no idea why only files below 12MB work, but thats just something i noticed. Also some kind of indicator for the upload would be great. Something like "10MB/120MB uploaded" which then updates every 0.5 seconds or so (just an idea from my side, but maybe there are other options aswell). That might be helpful

**Comment by BrunoBernardino** [comment]
Thanks, I'll have to investigate what's up because it shouldn't be limited. As for the progress indicator, I'd like to eventually add one, yeah, but definitely not a must-have for me (unlike to mobile auto-upload). Do you have any proxy in front of the container that could be limiting things? Also, are there no errors in the `docker compose logs`?

**Comment by Joly0** [comment]
Hey, so my proxy is nginx proxy manager (so basically nginx) but with default settings, shouldnt be an issue. Otherwise no, there is nothing in the logs. I can just see, that bewcloud loaded, and that i went to the files page, thats all. Nothing more. Is there like a debug mode for additional logging?

**Comment by BrunoBernardino** [comment]
I'm guessing if you see nothing, it's getting stuck in Nginx before it reaches bewCloud, but I can't be certain. I personally use Caddy, but have you tried increasing the [`client_max_body_size`](https://stackoverflow.com/a/26717238) in your nginx proxy entry?

**Comment by Joly0** [comment]
Yep, tried that setting in nginx already, didnt help. I can see, that bewcloud is trying to load something, but nothing happens then. The loading circle in the bewcloud ui just keeps loading for a moment

**Comment by BrunoBernardino** [comment]
Alright, I'll have to set aside some time to test it. For now, I'm not sure if you've found a workaround, but you can simply `cp` or `scp` the files into `./data-files/<your-user-id>` directly (that's what I did as it was a few GB to move over) as it just shows the file system, there's no database indexing to worry about.

**Comment by BrunoBernardino** [comment]
I've just tried with a local (not proxied) and remote instance (proxied via Caddy, without any file/upload/body specific settings) and had no trouble uploading a 50MB file. Do you see any error in your browser console (or network tab)?

**Comment by BrunoBernardino** [comment]
Let me know if you can get any more info here.

**Comment by Joly0** [comment]
Ok, so i tried looking into everythign that might cause nginx to give such an error, but i couldnt find anything. Otherwise i tried setting up caddy, but i struggled getting it to work with my certificates, so i will just leave it as is with nginx. I tried using a static ip as the base url, but couldnt login, i just end up on the login page again. I tried setting base_url to "http://192.168.178.220:8005", so that should be right
